pkg/multiTenancyPlugins/authorization/volumes.go

```diff
@@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec *httptest.ResponseRecorder) []byte {
 	}
 	return buf.Bytes()
 }
-func hostFSMountCheck(tenantId string, volumeBindings[] string) (error) {
-	for _,b := range volumeBindings {
-		if index := strings.Index(b,":"); index > -1 {
-		  v := b[0:index]
-		  if strings.Contains(v,"/") {
-			return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
-		  }
+func hostFSMountCheck(tenantId string, volumeBindings []string) error {
+	for _, b := range volumeBindings {
+		if index := strings.Index(b, ":"); index > -1 {
+			v := b[0:index]
+			if strings.Contains(v, "/") {
```

Comment:
> From: Doron Podoleanu <notifications@github.com>
> To: swarm-hooks/swarm <swarm@noreply.github.com>
> Cc: Kenneth Nagin/Haifa/IBM@IBMIL, Author <author@noreply.github.com>
> Date: 20/07/2016 10:46 AM
> Subject: Re: [swarm-hooks/swarm] Volume binding (#46)
> 
> In pkg/multiTenancyPlugins/authorization/volumes.go:
> > @@ -40,21 +38,21 @@ func filterVolumes(r *http.Request, rec 
> *httptest.ResponseRecorder) []byte {
> >     }
> >     return buf.Bytes()
> >  }
> > -func hostFSMountCheck(tenantId string, volumeBindings[] string) 
(error) {
> > -   for _,b := range volumeBindings {
> > -      if index := strings.Index(b,":"); index > -1 {
> > -        v := b[0:index]
> > -        if strings.Contains(v,"/") {
> > -         return errors.New(HOST_FS_MOUNT_NOT_ALLOWED)
> > -        }
> > +func hostFSMountCheck(tenantId string, volumeBindings []string) error 
{
> > +   for _, b := range volumeBindings {
> > +      if index := strings.Index(b, ":"); index > -1 {
> > +         v := b[0:index]
> > +         if strings.Contains(v, "/") {

> Does that cover all cases? What if there is something like dir1/dir2
dir1/dir2 is an illegal combination:
nagin@rcc-hrl-kvg-558:~$ sudo docker run  -v tmp/flavors:/tmp busybox top
docker: Error response from daemon: create tmp/flavors: volume name 
invalid: "tmp/flavors" includes invalid characters for a local volume 
name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed.
See 'docker run --help'.